src/camera-controls.js

```diff
@@ -553,27 +612,27 @@ export default class CameraControls {
 
 	}
 
-	reset( enableTransition ) {
+	getTarget() {
 
-		this._targetEnd.copy( this._target0 );
-		this._sphericalEnd.setFromVector3( _v3.subVectors( this._position0, this._target0 ) );
-		this._sphericalEnd.theta = this._sphericalEnd.theta % ( 2 * Math.PI );
-		this._spherical.theta    = this._spherical.theta    % ( 2 * Math.PI );
+		return this._targetEnd.clone();
```

Comment:
How about taking out-vector3 for the argument?

```
getTarget( out ) {

  const _out = out.isVector3 ? out : new THREE.Vector3();
  return _out.copy( this._targetEnd )
```
}

Reply:
AAAA cool!! I'm still pretty bad at such Three.js-friendly API 😅
I'll do this tmrw